molo/surveys/tests/test_rules.py

```diff
@@ -302,6 +302,16 @@ def test_call_test_user_without_user_or_request(self):
         rule = SurveyResponseRule(survey=self.survey)
         self.assertFalse(rule.test_user(None))
 
+    def test_get_column_header(self):
+        rule = SurveyResponseRule(survey=self.survey)
+        self.assertEqual(rule.get_column_header(), 'Other Survey')
+
+    def test_get_user_info_returns_submission_date(self):
+        current_date = datetime.datetime.now().strftime("%Y-%m-%d %H:%M")
+        self.submit_survey(self.survey, self.user)
+        rule = SurveyResponseRule(survey=self.survey)
+        self.assertEqual(rule.get_user_info_string(self.user), current_date)
```

Comment:
It might be worth bearing in mind that this test will fail intermittently if line 310 runs in a different minute to line 311. However I think the chance of that happening is super super slim, but it will be annoying to debug.

I'm hyper-aware of this kind of thing because I pushed a test yesterday that did a very similar thing in a much worse way :)